test/gui/tst_vfs/test.feature

```diff
@@ -38,19 +38,25 @@ Feature: Enable/disable virtual file support
         Given user "Alice" has been created on the server with default attributes and without skeleton files
         And user "Alice" has uploaded file with content "sample file" to "sampleFile.txt" in the server
         And user "Alice" has uploaded file with content "lorem file" to "lorem.txt" in the server
+        And user "Alice" has uploaded file with content "test file" to "testFile.txt" in the server
```

Comment:
This test was supposed to be on a virtual file so, a new test file was added because the previous two files were downloaded during the copy-paste operation.